test/gie/builtins.gie

```diff
@@ -4897,6 +4897,61 @@ expect  -0.001790493 0.000895247
 accept  -200 -100
 expect  -0.001790493 -0.000895247
 
+===============================================================================
+Tobler-Mercator
+	Cyl, Sph
+===============================================================================
+
+-------------------------------------------------------------------------------
+operation +proj=tobmerc +ellps=sphere
+-------------------------------------------------------------------------------
+tolerance 0.1 mm
```

Comment:
can you test the error case of latitude = 90 ?

Reply:
Done